elfeed.el

```diff
@@ -513,7 +528,8 @@ Only a list of strings will be returned."
                  (or (elfeed-feed-url url-or-feed)
                      (elfeed-feed-id url-or-feed))
                url-or-feed)))
-    (mapcar #'elfeed-keyword->symbol (cdr (assoc url elfeed-feeds)))))
+    (unless (elfeed-protocol-feed-p url)
+      (mapcar #'elfeed-keyword->symbol (cdr (assoc url elfeed-feeds))))))
```

Comment:
Make properties in protocol url like `("proto+https://user2@myhost.com" :prop1 "xxx" :prop2 "xxx")` works